warp/Network/Wai/Handler/Warp/IO.hs

```diff
@@ -10,7 +8,7 @@ import Network.Wai.Handler.Warp.Buffer
 import Network.Wai.Handler.Warp.Imports
 import Network.Wai.Handler.Warp.Types
 
-toBufIOWith :: Int -> IORef WriteBuffer -> (ByteString -> IO ()) -> Builder -> IO Int
+toBufIOWith :: Int -> IORef WriteBuffer -> (ByteString -> IO ()) -> Builder -> IO Integer
```

Comment:
Isn't `Int64` good enough?

Reply:
Would also be fine, but then we'd have to `fromIntegral` it to `Integer` at the next step anyway. So thought, why not just do `Integer` immediately.